src/main/resources/application.yml

```diff
@@ -8,3 +8,6 @@ spring:
     password: root
   Jackson:
     default-property-inclusion: non_null
+  mvc:
+    pathmatch:
+      matching-strategy: ant_path_matcher
```

Comment:
foi adicionado esse campo no application, pois quando foi implementado o swagger começou a dar pau ao rodar o projeto. Tendo em vista que a versão do swagger não suportava ao padrão do spring. 

Link da ajuda: https://stackoverflow.com/questions/40241843/failed-to-start-bean-documentationpluginsbootstrapper-in-spring-data-rest